tasks/level-1/1.8.yml

```diff
@@ -8,6 +8,7 @@
     name: "*"
     state: latest
     exclude: nginx*
+    update_cache: true
```

Comment:
the docs say the options are `yes` or `no` does `true` work also?

Reply:
Yeah, `true` and `false` works, and is our preferred way of denoting booleans. 

To add some more color to that: YAML--and Ansible by extension-- accepts a pretty wide variety of boolean-like values: `yes/no`, `true/false`, `True/False`, `y/n`, `1/0`, and there are probably a few more. We prefer to standardize on one, and since `true/false` happens to be the form used in most of the languages that we use here, it's the one we chose.